easybuild/easyconfigs/m/MultiQC/MultiQC-0.9-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,32 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Adam Huffman
+# The Francis Crick Institute
+easyblock = 'PythonPackage'
```

Comment:
several required Python packages are not provided by the `Python` installation, e.g. `click`, `multiqc`, `PyYaml,` ...